features/calculateTotalPurchase.feature

```diff
@@ -0,0 +1,28 @@
+Feature: Estatísticas
+
+    Scenario: Ver estatÍstica geral
+        Given que o usuário com CPF “00000000013” é o administrador
+        And o usuário está na página de administrador
+        And A média, o máximo e o mínimo do valores das compras (no tempo) são respectivamente "15000", "20000" e "10000"

```

Comment:
Just to make it easier to understand, you can replace it with:

```
And as métricas de valores "dos pedidos no tempo" são "15000" para "média", "20000" para "valor máximo" e "10000" para "valor mínimo"
```

Reply:
I chose the word "métricas" to refer to the statistics values, but you can choose another word. If you choose a different word, please remember to use it in all scenarios. All steps of this type must be identical, with the only difference being the parameters (the strings you passed such as "15000", "média", "valor máximo"...)

Reply:
When it comes to verifying that these values are correct, you can make a request to the backend. You can use [this](https://docs.cypress.io/api/commands/request) to receive the data in Cypress (test framework).

Reply:
If you have any doubts, you can always send me a message, and I will get back to you as soon as possible.